VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/hearing-controls/hearing-controls-base.component.spec.ts

```diff
@@ -141,6 +143,7 @@ describe('HearingControlsBaseComponent', () => {
 
         featureFlagServiceSpy = jasmine.createSpyObj<FeatureFlagService>('FeatureFlagService', ['getFeatureFlagByName']);
         featureFlagServiceSpy.getFeatureFlagByName.and.returnValue(of(true));
+        ldSpy.getFlag.and.returnValue(of(true));
```

Comment:
```suggestion
        launchDarklyServiceSpy.getFlag.withArgs(FEATURE_FLAGS.wowzaKillButton).and.returnValue(of(true));
```

to avoid blanket flag settings which could interfere with other feature flag calls